From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Islam Amer <islam.amer@jollamobile.com>
Date: Wed, 12 Nov 2014 18:10:53 +0200
Subject: [PATCH] Tracker config overrides

[schema] change default miner initial sleep to 30 seconds to not hammer the system during bootup
Set the default delay for GraphUpdated to 350 instead of upstream default of 1000. Helps fixing JB#11570
[schema] change default fs miner settings to match nemo defaults
Changing the default settings in the schema should hopefully reduce issues when user resets tracker with tracker-control or similar
[defaults] match default settings with oneshot configs for throttle
[schema] change default fs miner settings to better handle android storage
[schema] exclude Android folder from indexing
Harmonize ignore folders
---
 ...g.freedesktop.Tracker.Miner.Files.gschema.xml | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/data/org.freedesktop.Tracker.Miner.Files.gschema.xml b/data/org.freedesktop.Tracker.Miner.Files.gschema.xml
index 044173002..f5666b0c8 100644
--- a/data/org.freedesktop.Tracker.Miner.Files.gschema.xml
+++ b/data/org.freedesktop.Tracker.Miner.Files.gschema.xml
@@ -23,14 +23,14 @@ Boston, MA  02110-1301, USA.
       <summary>Initial sleep</summary>
       <description>Initial sleep time, in seconds.</description>
       <range min="0" max="1000"/>
-      <default>15</default>
+      <default>30</default>
     </key>
 
     <key name="throttle" type="i">
       <summary>Throttle</summary>
       <description>Indexing speed, the higher the slower.</description>
       <range min="0" max="20"/>
-      <default>0</default>
+      <default>10</default>
     </key>
 
     <key name="low-disk-space-limit" type="i">
@@ -48,7 +48,7 @@ Boston, MA  02110-1301, USA.
 	disables it entirely.
       </description>
       <range min="-2" max="365"/>
-      <default>-1</default>
+      <default>0</default>
     </key>
 
     <key name="removable-days-threshold" type="i">
@@ -59,7 +59,7 @@ Boston, MA  02110-1301, USA.
 	maximum is 365.
       </description>
       <range min="0" max="365"/>
-      <default>3</default>
+      <default>30</default>
     </key>
 
     <key name="enable-monitors" type="b">
@@ -71,7 +71,7 @@ Boston, MA  02110-1301, USA.
     <key name="index-removable-devices" type="b">
       <summary>Index removable devices</summary>
       <description>Set to true to enable indexing mounted directories for removable devices.</description>
-      <default>false</default>
+      <default>true</default>
     </key>
 
     <key name="index-optical-discs" type="b">
@@ -132,13 +132,13 @@ Boston, MA  02110-1301, USA.
     <key name="ignored-directories" type="as">
       <summary>Ignored directories</summary>
       <description>List of directories to avoid</description>
-      <default>[ 'po', 'CVS', 'core-dumps', 'lost+found' ]</default>
+      <default>[ 'po', 'CVS', 'core-dumps', 'lost+found', '$HOME/android_storage/Android' ]</default>
     </key>
 
     <key name="ignored-directories-with-content" type="as">
       <summary>Ignored directories with content</summary>
-      <description>Avoid any directory containing a file blocklisted here</description>
-      <default>[ '.trackerignore', '.git', '.hg', '.nomedia' ]</default>
+      <description>Avoid any directory containing a file blacklisted here</description>
+      <default>[ '.trackerignore', 'backup.metadata', '.nomedia' ]</default>
     </key>
   </schema>
 </schemalist>
